utilities/examine_focus_w_grid_scipy.py

```diff
@@ -76,10 +76,9 @@ def variance_of_laplacian(image):
 rgb_weights = [0.299, 0.587, 0.114]
 
 # Initialize camera and grab a reference to the raw camera capture
-camera = PiCamera()
+camera = PiCamera(resolution=(640, 480),
```

Comment:
Is there a functional reason these are being set in the constructor (as opposed to after the fact as the code had it)? Does seem cleaner (but maybe, in that case, rotation should also be set in the constructor, too) but as this is the only modification to this file, one could argue it would be better not to make the change for only refactoring reasons so that the PR remains tight.

Reply:
The reason that I set the resolution within the constructor is that in the documentation, it states the following: 

"The sensor_mode, resolution, framerate, framerate_range, and clock_mode parameters provide initial values for the sensor_mode, resolution, framerate, framerate_range, and clock_mode attributes of the class (these attributes are all relatively expensive to set individually, hence setting them all upon construction is a speed optimization). Please refer to the attribute documentation for more information and default values." [Link to PiCamera API](https://picamera.readthedocs.io/en/release-1.13/api_camera.html)

Reply:
That seems like a quite good reason to make that change.